modules/codegen/src/smithy4s/codegen/SmithyToIR.scala

```diff
@@ -59,13 +60,15 @@ private[codegen] class SmithyToIR(model: Model, namespace: String) {
 
   def allDecls = allShapes
     .filter(_.getId().getNamespace() == namespace)
-    .map(_.accept(toIRVisitor))
+    .map(_.accept(toIRVisitor(renderAdtMemberStructures = false)))
```

Comment:
This is how I went about preventing the adtMember structures from being rendered at the top level (like they normally are)